buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -1,27 +1,5 @@
 {
   "build": [
-    {
-      "environment": {},
-      "enabled": true,
-      "continueOnError": false,
-      "alwaysRun": false,
-      "displayName": "Run script $(VS140COMNTOOLS)\\VsDevCmd.bat",
-      "timeoutInMinutes": 0,
-      "condition": "ne(variables['PB_SkipTests'], 'true')",
-      "refName": "Task1",
-      "task": {
-        "id": "bfc8bf76-e7ac-4a8c-9a55-a944a9f632fd",
-        "versionSpec": "1.*",
-        "definitionType": "task"
-      },
-      "inputs": {
-        "filename": "$(VS140COMNTOOLS)\\VsDevCmd.bat",
-        "arguments": "",
-        "modifyEnvironment": "true",
-        "workingFolder": "",
-        "failOnStandardError": "false"
-      }
-    },
```

Comment:
With this removed, how will msbuild be found (there is an explicit msbuild call below)

Reply:
This is the only dependency that requires the toolset backed in. I seemed to have missed it, the tools directory should exist at this point to allow us to use Tools\msbuild.

Thoughts?

Reply:
@jashook I think that will work.  I'd check that there isn't another explicit call to msbuild in another json file if you haven't already.

Reply:
It also may be changeable to an msbuild task.  @MattGal May know why it wasn't in the first place.

Reply:
ping @MattGal